Readme.md

```diff
@@ -26,12 +26,13 @@ $ cd gcc
 $ sudo apt install flex libmpfr-dev libgmp-dev libmpc3 libmpc-dev
 $ ./configure \
    --enable-host-shared \
-   --enable-languages=jit \
+   --enable-languages=jit,c,c++ \
```

Comment:
The C and C++ frontends are not necessary right?

Reply:
They are for running libgccjit tests, so perhaps we should mention that.

Reply:
I'm currently checking without the C, however the C++ one is required for tests.